test/src/main/java/org/apache/accumulo/test/functional/CompactionIT.java

```diff
@@ -159,6 +165,77 @@ public void testBadSelector() throws Exception {
     }
   }
 
+  @Test
+  public void testDeleteTableAbortsCompaction() throws Exception {
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+      final String tableName = getUniqueNames(1)[0];
+      c.tableOperations().create(tableName);
+      c.tableOperations().setProperty(tableName, Property.TABLE_MAJC_RATIO.getKey(), "5.0");
```

Comment:
```suggestion
      var ntc = new NewTableConfiguration()
          .setProperties(Map.of(Property.TABLE_MAJC_RATIO.getKey(), "5.0"));
      c.tableOperations().create(tableName, ntc);
```

Could use `NewTableConfiguration` here